src/java/com/sarality/app/view/list/BaseViewRenderer.java

```diff
@@ -0,0 +1,20 @@
+package com.sarality.app.view.list;
+
+import android.content.Context;
+import android.view.View;
+
+/**
+ * Base View Renderer to render any kind of view
+ *
+ * @author sunayna@ (Sunayna Uberoy)
+ */
+public abstract class BaseViewRenderer<T> implements ViewRenderer<T> {
+  protected final Context context;
+  protected final View view;
+
+  public BaseViewRenderer(Context context, View view) {
```

Comment:
Makes sense. If and when you do add it, we may also want to add the ability to registerActions and setupActions in the ViewRenderer.